lms/djangoapps/teams/csv.py

```diff
@@ -446,6 +436,30 @@ def remove_user_from_team_for_reassignment(self, row):
                         # to readd the user, null out the team name
                         row[ts_id] = None
 
+    def _remove_user_from_teamset_and_emit_signal(self, user_id, ts_id, course_id):
+        """
+        If a team membership exists for the specified user, in the specified course and teamset, delete it.
+        This removes the user from the team.
+        Then, emit an event.
+
+        If the membership doesn't exist, don't emit the event and instead raise CourseTeamMembership.DoesNotExist
+        """
+        membership = CourseTeamMembership.objects.select_related('team').get(
```

Comment:
Just checking that the CourseTeamMembership.DoesNotExist is raised from the select_related.get(...) call?

Reply:
yeah, it's raised from the get(), the select_related() is just to grab the team as well if it does exist